tests/Common/AssignmentsStorageTestTrait.php

```diff
@@ -5,10 +5,31 @@
 namespace Yiisoft\Rbac\Tests\Common;
 
 use Yiisoft\Rbac\Assignment;
+use Yiisoft\Rbac\AssignmentsStorageInterface;
 use Yiisoft\Rbac\Item;
+use Yiisoft\Rbac\ItemsStorageInterface;
+use Yiisoft\Rbac\Permission;
+use Yiisoft\Rbac\Role;
+use Yiisoft\Rbac\Tests\Support\FakeAssignmentsStorage;
+use Yiisoft\Rbac\Tests\Support\FakeItemsStorage;
 
 trait AssignmentsStorageTestTrait
 {
+    private ?ItemsStorageInterface $itemsStorage = null;
+    private ?AssignmentsStorageInterface $storage = null;
+
+    protected function setUp(): void
+    {
+        $this->populateItemsStorage();
+        $this->populateStorage();
```

Comment:
```suggestion
        $this->populateAssignmentsStorage();
```

And other variables, methods, properties that keep assignments storage rename from `storage` to `assignmentsStorage`. I understand, that "storage" into `AssignmentsStorageTestTrait` means "assignments storage", but it's still confusing and not immediately clear.

Similar in `ItemsStorageTest` for items storage.